models/user_model.py

```diff
@@ -93,6 +94,10 @@ def delete_user(username):
         db.session.commit()
         return done
 
+    def update_last_login(self):
```

Comment:
New function update_last_login() needs unit tests to verify the datetime update and database commit functionality.